src/main/java/core/selenium/WebDriverManager.java

```diff
@@ -50,11 +51,13 @@ private WebDriverManager() {
         driver = BrowserFactory.getBrowser(browserType).getDriver();
         timeOutInSeconds = Integer.parseInt(WebDriverConfig.getInstance().getExplicit());
         wait = new WebDriverWait(driver, timeOutInSeconds);
-        driver.manage().timeouts().implicitlyWait(Integer.parseInt(WebDriverConfig.getInstance().getImplicit()), TimeUnit.SECONDS);
+        driver.manage().timeouts().implicitlyWait(Integer.parseInt(WebDriverConfig.
+                getInstance().getImplicit()), TimeUnit.SECONDS);
     }
```

Comment:
In the description of the class is the constructor.... of what? please write clear

Reply:
That is write in third person.